store/tikv/mock-tikv/pd.go

```diff
@@ -70,6 +70,11 @@ func (c *pdClient) GetRegionByID(ctx context.Context, regionID uint64) (*metapb.
 }
 
 func (c *pdClient) GetStore(ctx context.Context, storeID uint64) (*metapb.Store, error) {
+	select {
+	case <-ctx.Done():
+		return nil, ctx.Err()
+	default:
+	}
```

Comment:
This is the mock client behavior, it **should** check ctx.Done() to confirm to the real client behavior.
`loadStoreAddr` could check ctx.Done() but that's optional.